portfolio/src/main/webapp/script.js

```diff
@@ -35,9 +35,41 @@ async function helloName() {
   });
 }
 
+/**fetches comments from the server and adds them to the DOM*/
+async function loadComments() {
+  fetch('/data').then(response => response.json()).then((comments) => {
+    console.log(comments);
```

Comment:
Again with the previous comment about debug code not usually going into the main codebase.